_layouts/bare.html

```diff
@@ -21,8 +21,25 @@
     {% include nav.html %}
   </div>
 
-  <div class="bare-content" role="main" itemscope itemprop="mainContentOfPage">
-    {{ content }}{% include edit_link.html %}
+  <div ng-app="hubSearch" ng-controller="SearchController" class="bare-content" role="main" itemscope itemprop="mainContentOfPage">
+    {% raw %}
+      <div>
+        <live-search id="search1" type="text"
+          live-search-callback="searchCallback"
+          live-search-item-template="<a href='{{result.page.url}}'>{{result.page.title || result.page.url}}</a>"
```

Comment:
What should be in there? Seems like it's not necessary since the text is displayed:

http://stackoverflow.com/questions/5997371/is-the-title-attribute-for-links-required-for-508-compliance

/cc @nickbristow